### DISCUSSION TITLE
有没有办法让expand和collapse元素时，画面整体不要移动

### DISCUSSION THREAD

**Discussion by ksgfk** [antvis/G6#6523] [Q&A]
可复现的地址： [https://stackblitz.com/edit/p5kmma-ym9rvv?file=index.js](https://stackblitz.com/edit/p5kmma-ym9rvv?file=index.js)

https://github.com/user-attachments/assets/f0a05056-97fe-47e0-bc24-5175dc8eca78

可以看到无论是展开还是收起，画面都会自己移动到中心点，我希望有一种办法让移动前和移动后画面保持不动

**Comment by Aarebecca** [comment]
这是一个已知问题，会尽快修复
